test/integration/targets/ansible-vault/runme.sh

```diff
@@ -390,11 +380,14 @@ echo "The stdout log had 'New Vault password' in it and it is not supposed to. r
 
 # edit a 1.2 format with vault id, should keep vault id and 1.2 format
 EDITOR=./faux-editor.py ansible-vault edit "$@" --vault-id vault_password@vault-password "${TEST_FILE_EDIT2}"
-head -1 "${TEST_FILE_EDIT2}" | grep "${FORMAT_1_2_HEADER};vault_password"
+# shell check ignored as $ANSIBLE_VAULT not a shell var
+# shellcheck disable=SC2016
+head -1 "${TEST_FILE_EDIT2}" | grep -E '$ANSIBLE_VAULT;1.2;.+;vault_password'
```

Comment:
Shouldn't `$` be escape in this regex, though? Just like it used to be in `$FORMAT_1_2_HEADER`. Otherwise, it'd be expecting EOL in that place IIUC.
```suggestion
head -1 "${TEST_FILE_EDIT2}" | grep -E '\$ANSIBLE_VAULT;1.2;.+;vault_password'
```

Reply:
yep, fixed but not pushed as i was asked to not push while it was being reviewed